test/conftest.py

```diff
@@ -107,37 +107,31 @@ def _sql_session_rollback(self, attr):
 
 
 def pytest_collection_modifyitems(config, items):
+    # add pytest markers for tests that are not explicitly marked but include some keywords
+    name_to_markers = {
+        "generator": [pytest.mark.generator],
+        "summarizer": [pytest.mark.summarizer],
+        "tika": [pytest.mark.tika, pytest.mark.integration],
+        "parsr": [pytest.mark.parsr, pytest.mark.integration],
+        "ocr": [pytest.mark.ocr, pytest.mark.integration],
+        "elasticsearch": [pytest.mark.elasticsearch],
+        "faiss": [pytest.mark.faiss],
+        "milvus": [pytest.mark.milvus, pytest.mark.milvus1],
+        "weaviate": [pytest.mark.weaviate],
+        "pinecone": [pytest.mark.pinecone],
+        # FIXME GraphDB can't be treated as a regular docstore, it fails most of their tests
+        "graphdb": [pytest.mark.integration],
+    }
```

Comment:
Thanks for getting rid of the `if/else`s, I hope we can get rid of this entirely by reviewing our markers' usage at some point